x/stake/validator_test.go

```diff
@@ -148,7 +148,7 @@ func TestUpdateStatus(t *testing.T) {
 // TODO refactor this random setup
 
 // generate a random validator
-func randomValidator(r *rand.Rand) Validator {
+func randomValidator(r *rand.Rand, i int) Validator {
```

Comment:
previous random validator function would create multiple vals with all the same addrs and pks. This now will use different ones